composer.json

```diff
@@ -42,9 +38,8 @@
         "willdurand/negotiation": "^2.3"
```

Comment:
This dependency is not compatible with PHP 8 - therfore this patch isn't as well.

It declares a class name which is a keyword in PHP 8: https://github.com/willdurand/Negotiation/blob/2.x/src/Negotiation/Match.php

It must be `^3.0` to support PHP 8 - See willdurand/Negotiation#106